### PR TITLE
[services] don't copy fits on repeat runs

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

**Comment by acozine** [comment]
Does this approach work to avoid copying all the fits-related files, or does it just skip copying fits.sh?

**Comment by dunn** [comment]
The entire task is skipped: http://docs.ansible.com/ansible/shell_module.html

**Comment by acozine** [comment]
I've used this role for fits upgrades in the past; that wouldn't work with this change. How often and for what reasons are you re-running the playbook/roles/tasks?

**Comment by dunn** [comment]
I usually only have to re-run when it fails; yesterday I was doing a lot of runs while testing the ffmpeg changes so I started looking for anything I could speed up.

Any easy way we can check the version of FITS and only run the task if it's out of date? `fits.sh` is just a simple wrapper without any version info as far as I can see.

**Comment by acozine** [comment]
We could create fits.sh.{{ fits_ver }} and then symlink “fits” and “fits.sh” to the version-stamped file. Then the “creates” parameter would allow upgrades. 

As a non-code fix for the re-run issue, you can use the start-at-task flag and/or comment out roles in the main playbook to focus your post-failure test runs. Either way is more efficient than re-running the entire playbook every time you test a change.

**Comment by dunn** [comment]
Updated, how does that look?

**Comment by dunn** [comment]
> As a non-code fix for the re-run issue, you can use the start-at-task flag and/or comment out roles in the main playbook to focus your post-failure test runs.

Oh yeah, I keep forgetting about that.  One downside of using my fancy wrapper script :(

File: roles/services/tasks/fits.yml [hunk 1]
```diff
@@ -15,10 +15,17 @@
   become: yes
   file: path={{ install_path }}/fits-{{ fits_ver }}/fits.sh mode=0755
 
+- name: add version to fits wrapper
```
Comment (acozine) [inline roles/services/tasks/fits.yml:18]: I think you can combine adding the version number and copying the file in a single task.

Comment (acozine) [inline roles/services/tasks/fits.yml:18]: oh, wait, maybe not with the other files . . . . hm

**Pull request merged**